superset-frontend/src/features/alerts/types.ts

```diff
@@ -123,3 +123,25 @@ export interface AlertsReportsConfig {
   ALERT_REPORTS_DEFAULT_RETENTION: number;
   ALERT_REPORTS_DEFAULT_CRON_VALUE: string;
 }
+
+export type SectionValidationObject = {
+  status: boolean;
+  errors: string[];
+  name: string;
+};
+
+export interface ValidationObject {
+  generalSection: SectionValidationObject;
+  contentSection: SectionValidationObject;
+  alertConditionSection: SectionValidationObject;
+  scheduleSection: SectionValidationObject;
+  notificationSection: SectionValidationObject;
+}
+
+export enum Sections {
+  GENERAL = 'generalSection',
```

Comment:
We're very close to merging a [PR](https://github.com/apache/superset/pull/26875) which enforces PascalCase for enums. Could you rename those here so that we prevent conflicts?